posts/2023-05-02-23.0.0.4.adoc

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "ARM64 container images in Open Liberty 23.0.0.4"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: ARM64 container images in Open Liberty 23.0.0.4 - OpenLiberty.io
+seo-description: Open Liberty 23.0.0.4 provides better, more consistent logging for the features with which the runtime is running.  Joining the existing AMD64, PPC64LE, and S390X architectures, this release also provides container images for the ARM64 architecture.  This release also includes an important security vulnerability (CVE) fix and many notable bug fixes.
+blog_description: Open Liberty 23.0.0.4 provides better, more consistent logging for the features with which the runtime is running.  Joining the existing AMD64, PPC64LE, and S390X architectures, this release also provides container images for the ARM64 architecture.  This release also includes an important security vulnerability (CVE) fix and many notable bug fixes.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= ARM64 container images in Open Liberty 23.0.0.4
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 23.0.0.4 provides better, more consistent logging for the features with which the runtime is running.  Joining the existing AMD64, PPC64LE, and S390X architectures, this release also provides container images for the ARM64 architecture.  This release also includes an important security vulnerability (CVE) fix and many notable bug fixes.
+
+In link:{url-about}[Open Liberty] 23.0.0.4:
+
+* <<arm64, ARM64 container image>>
+* <<log, Consistently log features on server start>>
+//* <<api, Open Liberty API and SPI documentation>>
+* <<CVEs, Security Vulnerability (CVE) Fixes>>
+* <<bugs, Notable bug fixes>>
+
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23004+label%3A%22release+bug%22[23.0.0.4].
```

Comment:
Should this sentence clarify that the link goes to GitHub? It looks like we're linking to the bug fixes twice in a row because that's how the list ends too
```suggestion
View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23004+label%3A%22release+bug%22[23.0.0.4] on GitHub.
```

Reply:
I've been wondering if we just want to delete this one liner when we have the bugs section (and just keep it when we don't include the bugs section)